src/services/upload/videoMetadataService.ts

```diff
@@ -1,9 +1,15 @@
 import { NULL_EXTRACTED_METADATA } from 'constants/upload';
 import ffmpegService from 'services/ffmpeg/ffmpegService';
+import { ElectronFile } from 'types/upload';
 import { logError } from 'utils/sentry';
 
-export async function getVideoMetadata(file: File) {
+export async function getVideoMetadata(file: File | ElectronFile) {
     let videoMetadata = NULL_EXTRACTED_METADATA;
+    if (!(file instanceof File)) {
```

Comment:
can we just pass `ElectronFile` as `File` to `getVideoMetadata` ?

Reply:
I am not sure if we can, since the ffmpeg function uses a blob as an argument (which the File type satisfies).

Reply:
super nit: `file instanceOf ElectronFile` instead of `!(file instanceof File)`.  This will also call out this line whenever someone will look for reference of `ElectronFile`

Reply:
The `instanceof` needs a constructor for comparison which the `File` is, but `ElectronFile` is currently just a interface type.